examples/wadler_break_and_breakable/value.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative '../helper'
+
+# A value to be displayed in case no new line is needed can be specified.
```

Comment:
```suggestion
# A value to be displayed in case no new line is needed.
```

Reply:
But I don't get why we need to show this example or the one in `width.rb`. It seems useless info.